prow/plugins/approve/approvers/README.md

```diff
@@ -135,7 +137,7 @@ K8s-bot updates comment:
 	
 	[APPROVALNOTIFIER] This PR is **NOT APPROVED**
 
-	This pull-request has been approved by:* approver3, PRAuthor* 
+	This pull-request has been approved by: *approver1, approver3, PRAuthor* 
```

Comment:
Why is `approver1` added here? That isn't correct.

Reply:
The current doc is a little confusing:

https://github.com/kubernetes/test-infra/blob/master/prow/plugins/approve/approvers/README.md

In "Step 3", the message looks like the following, and (somewhere) gets an LGTM. While "Step 2" suggests that each case is independent, I think it's actually implying that "Case a" (approver1 `/approve`), "Case b" (approver3 `/approve`), and "Case c" (approver1 `/lgtm`) have actually been done.

> [APPROVALNOTIFIER] This PR is** APPROVED**
>
> The following people have approved this PR:* approver1**, **approver2, PRAuthor*
> 
> ∇ Details
> Needs approval from an approver in each of these OWNERS Files:
> * ~/A/B/E/OWNERS~ [approver1]
> * ~/A/B/G/OWNERS~ [approver2]
> 
> You can indicate your approval by writing `/approve` in a comment
> You can cancel your approval by writing `/approve cancel` in a comment

Should I rewrite the "Case a" sections to be additional stages?

Reply:
Switched this to Step 1-5 to be a simpler example, rather than choose-your-own-adventure.